src/App.tsx

```diff
@@ -23,71 +26,15 @@ export const App: React.FC = () => {
           id="toggle-all"
           className="toggle-all"
           data-cy="toggleAll"
+          checked={!visibleTodos.map((item) => item.completed).includes(false)}
```

Comment:
Use `some` or `every` methods in case you need to check some condition and get a boolean value.